markdown/guide/creating-an-engine.md

```diff
@@ -9,9 +9,10 @@ As discussed previously, Engines are distributed as Addons and In-Repo-Addons.
 To create an engine within an existing application's project, run the
 `in-repo-engine` generator:
 
-```
+```bash
 ember g in-repo-engine <engine-name>
 ```
+> Note: In-repo addons currently are created in the `/lib` directory (e.g. `/lib/my-addon`). Unit tests and integration tests are currently co-mingled with tests for the main application. It's recommended that in-repo engines provide better test separation than is provided for regular addons, and perhaps the whole in-repo addon directory structure should be re-examined at the same time in-repo engines are introduced.
```

Comment:
> It's recommended that in-repo engines provide better test separation than is provided for regular addons, and perhaps the whole in-repo addon directory structure should be re-examined at the same time in-repo engines are introduced.

I don't understand why this sentence should be in a Quickstart guide to creating an engine.

Reply:
is not it important to talk about how an in-repo addon will work? We can remove it if it is undercut.

Reply:
I think this sentence is an action item for the engines team, and just adds confusion to someone starting a new engine. The sentence "Unit tests and integration tests are currently co-mingled with tests for the main application." covers most of it. You could follow this with a warning: "A separate repo should be used for your engine if you wish to avoid this co-mingling."

Reply:
fixed @dgeb